os/drivers/lcd/lcd_dev.c

```diff
@@ -464,7 +467,13 @@ int lcddev_register(struct lcd_dev_s *dev)
 	lcd_init_put_image(dev);
 	sem_init(&lcd_info->sem, 0, 1);
 	if (dev->setpower) {
-		ret = dev->setpower(dev, CONFIG_LCD_MAXPOWER);
+		bool is_silent_mode;
+		is_silent_mode = silent_reboot_is_silent_mode();
+		if (is_silent_mode) {
+			ret = dev->setpower(dev, 0);
```

Comment:
If this case, don't call pm_suspend()

The pm_susepnd must be called only lcd turning on state.

If you make lcd turn off state, don't call pm_suspend or, need to release pm_suspend using pm_resume

Reply:
How about
```
sem_init(&lcd_info->sem, 0, 1);
is_silent_mode = silent_reboot_is_silent_mode();
if (!is_silent_mode) {
	lcd_init_put_image(dev);
	if (dev->setpower) {
		ret = dev->setpower(dev, CONFIG_LCD_MAXPOWER);
		if (ret != OK) {
			goto cleanup;
		}
#ifdef CONFIG_PM
		(void)pm_suspend(lcd_info->pm_domain);
#endif
	}
}
```

if you change default power 0 in mipi_lcd, we don't need to call setpower 0

Reply:
And if not silent mode, we need to add silent_reboot_lock(); here when lcd on